frontend/elements/src/widgets/nav/step-nav.ts

```diff
@@ -0,0 +1,62 @@
+import { LitElement, html, css, customElement, property } from 'lit-element';
+import { ifDefined } from 'lit-html/directives/if-defined';
+
+
+@customElement('step-nav')
+export class _ extends LitElement {
+
+    static get styles() {
+        return [css`
+            :host {
+                display: contents;
+            }
+            :host(:last-child) .line {
+                display: none;
+            }
+            .line {
+                width: 100%;
+                margin-top: 25px;
+                display: grid;
+            }
+            .line::after {
+                content: '';
+                display: inline-block;
+                background-color: var(--Light_Gray_1);
+                height: 2px;
+                width: calc(100% + 50px);
+                margin-left: -25px;
+            }
+            :host([completed]) .line::after {
+                background-color: var(--Dark_Green_1);
+            }
+        `];
+    }
+
+    @property({type: Number})
+    number: number = 1;
+
+    @property({type: String})
+    label: string = "";
+
+    @property({type: Boolean})
+    completed: boolean = false;
+
+    @property({type: Boolean})
+    active: boolean = false;
+
+    render() {
+        return html`
+            <button-circle
+                label="${this.label}"
+                color="${ifDefined(
+                    this.active ?  "blue"
+                        : this.completed ? "green"
```

Comment:
wasn't aware of ifDefined directive, cool stuff :)

Not for now - but in the future maybe we should have a utility function that wraps around it, maybe with an interface like [Ramda's cond](https://ramdajs.com/docs/#cond)